docs/tutorial_bracket_strategy.md

```diff
@@ -0,0 +1,224 @@
+
+The following script will place programmable orders for the pair DAI-WETH on a price range [150,260] on Rinkeby.
+You will invest 1000 [DAI](https://rinkeby.etherscan.io/address/0x5592EC0cfb4dbc12D3aB100b257153436a1f0FEa) and 5 [WETH](https://rinkeby.etherscan.io/address/0xc778417E063141139Fce010982780140Aa0cD5Ab).
+
+```ssh
+export NETWORK_NAME=rinkeby
+export GAS_PRICE_GWEI=1
+export MASTER_SAFE=<your master safe>
+export PK=<your private key of the proposer account>
+```
+
+```ssh
+cd dex-liquidity-provision
+yarn install
+npx truffle compile
+yarn run networks-inject
+npx truffle exec scripts/complete_liquidity_provision.js --targetToken=0 --stableToken=7 --lowestLimit=150 --highestLimit=260 --currentPrice=200 --masterSafe=$MASTER_SAFE --investmentTargetToken=5 --investmentStableToken=1000 --fleetSize=20 --network=$NETWORK_NAME
+```
+
+The script will first make some plausibility checks as the mentioned price check and that the boundaries - highestLimit and lowerLimit - are set reasonable for the current price.
+
+Then, it will create a transaction deploying new safes. All the address of the Gnosis-Safes will be printed on the screen.
+
+Afterwards, the transaction containing all the orders will be build.
+All the order placements will be bundled into one transaction, which will appear first in the Gnosis-Interface.
+Before you sign the transaction, make sure to check the correctness of the displayed orders of the script output.
+
+Also, the transaction sending the DAI and WETH into the brackets and depositing it on behalf of the brackets into the GP exchange contract is built in parallel.
+This is the second transaction send to the interface.
+
+If everything goes smoothly, you should see the two transaction initiations confirmed like that:
+
+```ssh
+
+==> Sending the order placing transaction to gnosis-safe interface.
+    Attention: This transaction MUST be executed first!
+
+Signing and posting multi-send transaction 0x007ecc58f9dc222c26215b260abc363e0308dfec539d7eeabb8a2c9f09397157 from proposer account 0x740a98F8f4fAe0986FB3264Fe4aaCf94ac1EE96f
+Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
+
+==> Sending the funds transferring transaction, please execute this transaction second
+
+Signing and posting multi-send transaction 0x09cb78a5a49f10305a2f108d45e8fa059e4231ebf219315c56409d67419d65fb from proposer account 0x740a98F8f4fAe0986FB3264Fe4aaCf94ac1EE96f
+Transaction awaiting execution in the interface https://rinkeby.gnosis-safe.io/app/#/safes/$MASTER_SAFE/transactions
+```
+
+Now, you should just follow the link and sign & execute the transaction with the lower transaction-ID first. It should look like this:
+
+![Read tokenAddressToIdMap](assets/tutorial_bracket_strategy_signing.png)
+
+For executing, just press the button _confirm_.
+
+If you are executing the second transaction with a lower gasPrice than the first one, then you can sign it immediately.
```

Comment:
Is this true? If one has lower gas price but requires less gas overall it might still be included in the block before the one with higher price.

Reply:
I don't believe so. Especially not now that we specify the nonce. It is not even possible to submit the second until the first has already been submitted.

Reply:
Yep, just tested, you are right! The second transaction cannot be executed before the first has been confirmed. Then the wording here should probably change.